src/main/java/net/dries007/tfc/client/IngameOverlays.java

```diff
@@ -312,4 +375,31 @@ public static boolean setup(ForgeIngameGui gui, Minecraft minecraft)
         }
         return false;
     }
+
+    private static int considerExperienceConfigs(int heightIn)
+    {
+        if (!TFCConfig.CLIENT.enableExperienceBar.get())
+        {
+            final LocalPlayer player = Minecraft.getInstance().player;
+            int heightOut;
+            switch (TFCConfig.CLIENT.disabledExperienceBarStyle.get())
+            {
+                case LEFT_HOTBAR -> heightOut = heightIn + 6;
+                case BUMP ->
+                {
+                    if (player != null && (player.fishing instanceof TFCFishingHook || player.isRidingJumpable()))
+                    {
+                        heightOut = heightIn;
+                    }
+                    else
+                    {
+                        heightOut = heightIn + 6;
+                    }
+                }
+                default -> heightOut = heightIn;
+            }
+            return heightOut;
```

Comment:
This can be quite simplified using a switch expression:
```suggestion
            return switch (TFCConfig.CLIENT.disabledExperienceBarStyle.get())
            {
                case LEFT_HOTBAR -> heightIn + 6;
                case BUMP ->player != null && (player.fishing instanceof TFCFishingHook || player.isRidingJumpable()) ? heightIn : heightIn + 6;
                default -> heightIn;
            }
```